chip/support/generate-test-descriptor

```diff
@@ -229,7 +229,7 @@ load_python()
 shutil.rmtree("/tmp/metadata-garbage")
 
 # Normalize/prettify names and create semantic version for sorting
-semantic_name_pattern = re.compile(r"(.*)_(\d+)_(\d+)(?:_(.*))?")
+semantic_name_pattern = re.compile(r"(.*)_(\d+)_(\d+(?:_\d+))(?:_(.*))?")
```

Comment:
Were you able to test this? That capturing group is parsed as an integer below, and I think the next one should retrieve this case as it accepts anything following “_“.

It may be that the structure of the test descriptor needs to change... I'm not sure if/how this script and/or the test harness handles multiple tests in one file